packages/cardano-services/src/NetworkInfo/DbSyncNetworkInfoProvider/DbSyncNetworkInfoProvider.ts

```diff
@@ -52,10 +50,10 @@ export class DbSyncNetworkInfoProvider extends DbSyncProvider implements Network
     super(db);
     this.#logger = logger;
     this.#cache = cache;
+    this.#cardanoNode = cardanoNode;
     this.#builder = new NetworkInfoBuilder(db, logger);
     this.#genesisDataReady = loadGenesisData(cardanoNodeConfigPath);
-    this.#epochPollInterval = epochPollInterval;
-    this.#cardanoNode = cardanoNode;
+    this.#epochPollService = EpochPollService.create(cache, () => this.#builder.queryLatestEpoch(), epochPollInterval);
```

Comment:
It would be better to inject this as part of `NetworkInfoProviderDependencies`

Reply:
https://github.com/input-output-hk/cardano-js-sdk/pull/370/commits/7c8cd1f53bc0bfecbb7fe3af6c33fa0b409288f6